Sources/QueuesFluentDriver/JobModelMigrate.swift

```diff
@@ -1,49 +1,32 @@
-import Foundation
-import Fluent
-import SQLKit
+import protocol FluentKit.AsyncMigration
+import protocol FluentKit.Database
+import protocol SQLKit.SQLDatabase
 
-public struct JobMetadataMigrate: Migration {
-    public init() {}
```

Comment:
I think we might still need this, otherwise I get this error:

'JobMetadataMigrate' initializer is inaccessible due to 'internal' protection level